### PR TITLE
[npud] Implement create and destroy context in core class

### PR DESCRIPTION
**Pull request by jyoungyun**
The Core function invokes the dev manager to get npu context and invokes the context manager to create npu daemon context. The npu daemon context internally contains npu context.

Signed-off-by: Jiyoung Yun <jy910.yun@samsung.com>

Draft PR: #9989

### CODE REVIEW

**Review by hseok-oh — APPROVED** [review]
LGTM

File: runtime/service/npud/core/Core.cc [hunk 1]
```diff
@@ -32,5 +33,44 @@ void Core::deinit() { _devManager->releaseModules(); }
...
+  {
+    VERBOSE(Core) << "Invalid context id" << std::endl;
+    // TODO Define CoreStatus
+    return 1;
+  }
+
+  int ret = _devManager->destroyContext(npuContext);
+  if (ret != NPU_STATUS_SUCCESS)
+  {
+    VERBOSE(Core) << "Failed to destroy npu context: " << ret << std::endl;
```
Comment (ragmani) [inline runtime/service/npud/core/Core.cc:68]: Is it OK not to return?

**Review by hseok-oh — APPROVED** [review]

**Review by ragmani — DISMISSED** [review]
LGTM

**Comment by hseok-oh** [comment]
@jyoungyun Please resolve conflict

**Review by hseok-oh — APPROVED** [review]

**Pull request merged**